rcl/test/rcl/allocator_testing_utils.h

```diff
@@ -87,6 +87,80 @@ set_failing_allocator_is_failing(rcutils_allocator_t & failing_allocator, bool s
   ((__failing_allocator_state *)failing_allocator.state)->is_failing = state;
 }
 
+typedef struct __time_bomb_allocator_state
```

Comment:
Don't use `__` as a prefix (I can try to find the docs, but those symbols are reserved according to some c++ guideline/standard/etc).

Reply:
I can change it. Please note though, I did it like this to match the convention used for the other failing allocator in this file, should I open an issue to change that?

Reply:
Removed with 9220f89

Reply:
>  I did it like this to match the convention used for the other failing allocator in this file, should I open an issue to change that?

If you want to fix that in another PR that's fine.